<ui version="4.0">
 <class>JoinMenu</class>
 <widget class="QWidget" name="JoinMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server info:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="serverip">
       <property name="text">
        <string>192.168.0.199</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="connectToIpLabel">
       <property name="text">
        <string>Connect to IP/Port:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="nickNameLineEdit">
       <property name="text">
        <string>Unnamed</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="nickNameLabel">
       <property name="text">
        <string>Player name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QListWidget" name="serverListWidget"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="joinOk">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="joinCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serverListWidget</tabstop>
  <tabstop>serverip</tabstop>
  <tabstop>joinOk</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>joinCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>joinCancel</sender>
   <signal>clicked()</signal>
   <receiver>JoinMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
